src/main/java/com/kodilla/good/patterns/food2Door/OrderDto.java

```diff
@@ -0,0 +1,29 @@
+package com.kodilla.good.patterns.food2Door;
+
+public class OrderDto {
+    String product;
+    private double quantity;
+
+    public OrderDto(String product, double quantity) {
+        this.product = product;
+        this.quantity = quantity;
+    }
+
+    public enum ShopTypeEnum {
+        EXTRA_FOOD_SHOP,
+        GLUTEN_FREE_SHOP,
+        HEALTHY_SHOP
+    }
```

Comment:
ShopTypeEnum jest klasa w klasie. Moze tutaj zostac, ale to dosc nietypowe. 
W OrderDto musisz dodac pole `private ShopTypeEnum shopTypeEnum`, zrob tez update konstruktora

Reply:
Wtedy ten kod
```
switch (orderDto.getShopTypeEnum()){
                case MEAT_SHOP:
                    meatShop.processOrder(numberOfUnits, product);
                default:
                    throw new ShopNotFoundException();
```
bedzie dzialac